backend/products/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+# Create your models here.
+
+class Product(models.Model):
+    name = models.CharField(max_length=200)
+    description = models.TextField(max_length=600, blank=True)
+    category = models.CharField(max_length=100)
+    primary_variant = models.OneToOneField(
+        'ProductVariation', on_delete=models.CASCADE, default=None, null=True)
+    
+    # Need to add Primary Variant and seller Id
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+    
+
+class ProductVariation(models.Model):
+    product_id = models.ForeignKey(Product, on_delete=models.CASCADE)
```

Comment:
let's add a related_name here

Reply:
Instead of product_id? We should add related_name?